Test/scripts/ci-integration.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -x
+
+trap '>&2 echo Error: Command \`$BASH_COMMAND\` on line $LINENO failed with exit code $?' ERR
+
+Test/scripts/install_magento.sh
```

Comment:
nit: could we calculate gitroot at the top and then call scripts relative to it?

this way i can run the script from any directory and itll work